tests/bsim/bluetooth/host/misc/sample_test/src/main.c

```diff
@@ -38,6 +60,7 @@ static const struct bst_test_instance entrypoints[] = {
 		.test_id = "dut",
 		.test_delete_f = test_end_cb,
 		.test_main_f = entrypoint_dut,
+		.test_args_f = test_args,
```

Comment:
@jori-nordic is there any reason why you suggested this being a test argument instead of being registered as a plain command line argument? (so it would be passed as `--log_level=4` instead of  `-argstest log_level 4`. You could have register & handled from the bt testlib directly instead of each test).

Reply:
I was wondering something similar. 
Might be possible to implement this in a more generic way rather than handling it for each test application

Reply:
The native simulator is meant to be really plug&play for this kind of things. So yep, you can do so. Something like adding in that bt testlib library the equivalent of 

https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/native/nrf_bsim/common/phy_sync_ctrl.c#L136
(but registering just one argument)
+ 
A hook like this
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/native/nrf_bsim/common/phy_sync_ctrl.c#L213
to have that command line argument registration function be called very early at runtime.
+
an equivalent to https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/native/nrf_bsim/common/phy_sync_ctrl.c#L100
to handle the argument when/if it is passed to the executable.

Reply:
Hi @aescolar it was just OTOH. I guess it's better to do that from the testlib, yeah. With a sensible default ofc.

Reply:
To be addressed in a future PR.